### ISSUE TITLE
Schema Compare - include/exclude multiple objects/rows

### ISSUE THREAD

**Issue by filipalai** [microsoft/azuredatastudio#19241]
It would be extremely convenient to be able to select multiple rows in the Schema Compare, and right-click and choose to Exclude or Include them (just like in Visual Studio). Also, the feature in Visual Studio where Added, Deleted and Changed object are grouped is very handy - as well as being able to right-click each group and select Exclude/Include.

**Comment by indexample** [comment]
for me, this is the main reason why im not using ADS anymore, back to visual studio schema compare. i dont want to manually uncheck hundreds of objects.

**Comment by tecodan** [comment]
> for me, this is the main reason why im not using ADS anymore. I dont want to manually uncheck (exclude) hundreds of objects.

What is the alternative tool you are using now?

**Comment by indexample** [comment]
Visual Studio has a schema compare option (or extension). there its easy to (un)check all, or all in the group Deletes, Changes, New objects.

**Comment by okarika** [comment]
Would be extremely useful...

**Comment by dp33Fr** [comment]
Please add 2 buttons « Select all » « unselect all »

**Comment by ltjkoomen** [comment]
Indeed, adding just the Select All / Unselect All buttons would save a huge amount of hassle already. 

The combination of:
- a large amount of changes
- users only wanting to apply a small subset of changes
- the Schema Compare tool asynchronously in the background trying to Re-select any items that are dependencies of items that are not even in the subset of interest (because users are still busy trying to deselect everything)

makes for a very frustrating experience, feels like playing whack-a-mole at times :(. 

Just being able to start from a completely unselected state would allow users to add the changes they want to apply and wait for the dependency manager to reselect anything that is required for the subset without getting in the way.
